src/driver/sqlite-abstract/AbstractSqliteQueryRunner.ts

```diff
@@ -456,26 +456,32 @@ export abstract class AbstractSqliteQueryRunner extends BaseQueryRunner implemen
     /**
      * Drops the columns in the table.
      */
-    async dropColumns(tableOrName: Table|string, columns: TableColumn[]): Promise<void> {
+    async dropColumns(tableOrName: Table|string, columns: TableColumn[]|string[]): Promise<void> {
         const table = tableOrName instanceof Table ? tableOrName : await this.getCachedTable(tableOrName);
 
         // clone original table and remove column and its constraints from cloned table
         const changedTable = table.clone();
-        columns.forEach(column => {
-            changedTable.removeColumn(column);
-            changedTable.findColumnUniques(column).forEach(unique => changedTable.removeUniqueConstraint(unique));
-            changedTable.findColumnIndices(column).forEach(index => changedTable.removeIndex(index));
-            changedTable.findColumnForeignKeys(column).forEach(fk => changedTable.removeForeignKey(fk));
+        columns.forEach((column: TableColumn|string) => {
+            const columnInstance = column instanceof TableColumn ? column : table.findColumnByName(column);
+            if (!columnInstance) return;
```

Comment:
We must throw an error here instead of silently break the operation:
```ts
if (!columnInstance)
   throw new Error(`Column "${column}" was not found in table "${table.name}"`);
```